backstop/tests/table.html

```diff
@@ -4396,5 +4396,29 @@ <h2>Empty state</h2>
         column.classList.remove('iui-reorder-column-left');
       };
     </script>
+
+    <script>
+      const setMargin = (actionsContainer) => {
+        const filterButton = actionsContainer.querySelector('.iui-filter-button');
+        if (!filterButton) {
+          return;
+        }
+        if (actionsContainer.clientHeight > 28) {
```

Comment:
Would need to check about that height (for different table sizes?)
Maybe we can think of some other identifier...

Reply:
Removed scripts